packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-tile.scss

```diff
@@ -0,0 +1,24 @@
+.msc-billing-tile-wrapper {
+  display: block;
+  text-decoration: none;
+  padding: var(--ods-size-01);
+
+  .billing-tile-content {
+    display: flex;
+    flex-direction: column;
+    min-width: 420px;
```

Comment:
**nitpick:** Instead of hard-coding the CSS in pixels, maybe we can think of creating a utility classes for super-components and reuse them to maintain consistency across the super-components.